src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.jelly

```diff
@@ -87,5 +87,10 @@
         <f:entry title="${%Max size of log file}" description="${%maxLogSize}">
             <f:textbox field="maxLogSize" />
         </f:entry>
+        <f:entry title="${%Set job description to failure description}"
+             description="${%Enabling this option will set the build's description to the failure descriptions as a convenience.}">
+            <f:checkbox name="buildDescriptionEnabled" checked="${it.buildDescriptionEnabled}"
+                  default="false"/>
+        </f:entry>
```

Comment:
You should use correct databinding, either set the `field` attribute on the entry or the checkbox, one of them should work, and remove the `checked` attribute.

Reply:
Done